app/src/main/java/com/android/sample/resources/C.kt

```diff
@@ -221,8 +224,36 @@ object C {
     const val RECIPE_PUBLISH_ERROR_MESSAGE = "Failed to publish recipe: %1\$s"
   }
 
+  object Values {
+    object RecipeOverview {
+      const val INITIAL_NUMBER_VALUE_PER_RECIPE = 1
```

Comment:
Hey what is this value exactly for ? Maybe just add a little comment...

Reply:
You're right, I've update it